packages/provider/src/transports/wallet-request-handler.ts

```diff
@@ -51,19 +50,18 @@ export class WalletRequestHandler implements ExternalProvider, JsonRpcHandler, P
 
   onConnectOptionsChange: ((connectOptions: ConnectOptions | undefined) => void) | undefined = undefined
 
-  public defaultNetworkId: number
-
   constructor(
     account: Account | null | undefined,
     prompter: WalletUserPrompter | null,
-    networks: NetworkConfig[],
-    defaultNetworkId?: string | number
+    networks: NetworkConfig[]
   ) {
     this.account = account
     this.prompter = prompter
     this.networks = networks
+  }
 
-    this.defaultNetworkId = defaultNetworkId ? this.findNetworkID(defaultNetworkId) : networks[0].chainId
+  defaultNetworkId(): number {
```

Comment:
rename to `defaultChainId`

I believe we use the term networkId or the NetworkId type to mean a string | number where the string could be a named identifier from the NetworkConfig:name field, ie. 'mainnet', 'polygon'

we should be clear that this is a chainId